building/windows.md

```diff
@@ -0,0 +1,182 @@
+functionality similar to a Linux distribution on Windows.
+
+* [Cygwin](https://cygwin.com/install.html)
+
+<details>
+<summary>Cygwin installation/update process (click to expand)</summary>
+&nbsp;
+
+* The installation process also serves as a package manager. Cygwin will ask of type of installation in our case it
+will be from internet
+
+![cygwin-download-source](_images/cygwin-download-source.png)
+
+* After that install process will ask about placement of root and local package directory
+
+![cygwin-root-dir](_images/cygwin-root-dir.png)
+![cygwin-local-directory](_images/cygwin-local-directory.png)
+
+* Cygwin has capability of using host's internet connection, but it needs to be configured with our preferences (In most
+scenario it will be System Proxy Settings)
+
+![cygwin-internet-connection](_images/cygwin-internet-connection.png)
+
+* Selection of mirror for packages downloads can be tricky, sometimes it needs to be changed for better results.
+
+![cygwin-mirror-selection](_images/cygwin-mirror-selection.png)
+
+* After selecting mirror we can select the packages we want. But in larger groups, this can be annoying. That's why we
+only select two elements (They will be installed with dependencies) which are mandatory to use package manager `apt-cyg`
+in Cygwin terminal.
+
+![cygwin-selecting-packages](_images/cygwin-selecting-packages.png)
+
+* Once these are chosen, proceed to run the installation process
+
+</details>
+  &nbsp;
+
+  <details>
+<summary>Installing apt-cyg and mandatory packages (click to expand)</summary>
+
+``` bash
+lynx -source rawgit.com/transcode-open/apt-cyg/master/apt-cyg > apt-cyg && \
+install apt-cyg /bin && \
+apt-cyg install libhidapi-devel libgccpp1 libtool \
+libusb-devel libvirt-client libvirt-devel \
+mingw64-i686-gcc-core mingw64-i686-gcc-g++ \
+mingw64-x86_64-gcc-core mingw64-x86_64-gcc-g++ \
+gcc-core gcc-g++ libclang-devel nano vim \
+patch picocom python3 python3-pip python3-sphinx \
+python3-devel qemu-integration symlinks \
+tty-clock usbutils virt-viewer make autoconf \
+automake cmake libc-client-devel git
+```
+
+  </details>
+  &nbsp;
+
+Before preforming next steps Cygwin needs to download all dependencies to packages that was installed.
+Launch Cygwin installation again, in packages select section we can see tons of libraries were auto added to list.
+All what we need is to execute this process.
+
+The next step is to prepare the genext2fs library and clone the repository. We need to make manually genext2fs library,
+because Cygwin doesn't support it, and it's mandatory in case of creating images for the ia32 system.
+
+To do this, paste the following into the Cygwin terminal:
+
+<details>
+<summary>Make genext2fs package and download phoenix-rtos (click to expand)</summary>
+
+``` bash
+git clone --recursive https://github.com/bestouff/genext2fs &&\
+cd genext2fs &&\
+./autogen.sh &&\
+./configure &&\
+make &&\

```

Comment:
This is strictly taken from application README because configure recognize virtual system and must check up capability of creating this type of files. I tried without autogen but configure failed (Probably some Cygwin related problems). 
And about make as author of library says:
![image](https://github.com/phoenix-rtos/phoenix-rtos-doc/assets/49757239/b4ae4305-5769-4630-a0cb-d1c3810f4709)